src/sndfile.h.in

```diff
@@ -92,6 +92,10 @@ enum
 	SF_FORMAT_GSM610		= 0x0020,		/* GSM 6.10 encoding. */
 	SF_FORMAT_VOX_ADPCM		= 0x0021,		/* OKI / Dialogix ADPCM */
 
+	SF_FORMAT_NMS_ADPCM_16	= 0x0022,		/* 16kbs NMS G721-variant encoding. */
```

Comment:
How wide should tabs be in libsndfile? These defines are inline when tabs ('\t') are at four characters. I've been using the vim settings of 'ts=4 sts=4 sw=4 noet' as that appeared to be correct.